autogluon/utils/tabular/ml/models/fastainn/hyperparameters/parameters.py

```diff
@@ -0,0 +1,48 @@
+from .....ml.constants import BINARY, MULTICLASS, REGRESSION
+
+# TODO this method is generalizable and potentially should be moved out into framework
+def get_param_baseline(problem_type, num_classes=None):
+    if problem_type == BINARY:
+        return get_param_binary_baseline()
+    elif problem_type == MULTICLASS:
+        return get_param_multiclass_baseline()
+    elif problem_type == REGRESSION:
+        return get_param_regression_baseline()
+    else:
+        return get_param_binary_baseline()
+
+
+def get_param_multiclass_baseline():
+    params = {
+        # See docs: https://docs.fast.ai/tabular.models.html
+        'layers': None,  # layers configuration; None - use model's heuristics
+        'emb_drop': 0.1,  # embedding layers dropout
+        'ps': [0.1],  # linear layers dropout
+        'bs': 256,  # batch size
+
+        # maximum learning rate for one cycle policy https://docs.fast.ai/train.html#fit_one_cycle
+        # One-cycle policy paper: https://arxiv.org/abs/1803.09820
+        'lr': 1e-2,
+        'epochs': 30,  # number of epochs
+        'metric': 'accuracy',
+
+        # Early stopping settings. See more details here: https://docs.fast.ai/callbacks.tracker.html#EarlyStoppingCallback
+        'early.stopping.min_delta': 0.0001,
+        'early.stopping.patience': 10,
+
+        # If > 0, then use LabelSmoothingCrossEntropy loss function for binary/multi-class classification;
+        # otherwise use default loss function for this type of problem
+        'smoothing': 0.0,
```

Comment:
Consider in future PR:  I'd suspect 'smoothing' =  ~0.01 may be a universally superior default option for most classification datasets

Reply:
I did some test on small automl benchmark tests and it dropped scores in most of the tests. 

|dataset          |metric |regular  |smoothing 0.01|improvement (positive == improved)|
|-----------------|-------|---------|--------|----------------------------------|
|Australian       |auc    |0.927844 |0.921902|-0.005942                         |
|blood-transfusion|auc    |0.710039 |0.645712|-0.064327                         |
|car              |logloss|0.0861503|0.403592|-0.3174417                        |
|christine        |auc    |0.796204 |0.73316 |-0.063044                         |
|cnae-9           |logloss|3.0715   |2.53242 |0.53908                           |
|credit-g         |auc    |0.827143 |0.768571|-0.058572                         |
|dilbert          |logloss|0.0989941|0.343822|-0.2448279                        |
|fabert           |logloss|0.923508 |1.29875 |-0.375242                         |
|jasmine          |auc    |0.85566  |0.852125|-0.003535                         |
|kc1              |auc    |0.798621 |0.80735 |0.008729                          |
|kr-vs-kp         |auc    |0.9991   |0.999413|0.000313                          |
|mfeat-factors    |logloss|0.071715 |0.857656|-0.785941                         |
|phoneme          |auc    |0.912575 |0.928611|0.016036                          |
|segment          |logloss|0.0849755|0.735826|-0.6508505                        |
|sylvine          |auc    |0.968537 |0.961135|-0.007402                         |
|vehicle          |logloss|0.367012 |0.548537|-0.181525                         |

Reply:
Interesting that's unexpected from my experience, maybe it's due to default hyperparams being designed around no-label smoothing.  Thanks for testing it though!  

I wonder if smoothing is very close to zero whether there is still a sizable performance-gap, which would indicate there's some implementation/hyperparameter issue... anyway seems best to simply leave it off for now, so it just remains a searchable hyperparameter.